docs/google-cloud.md

```diff
@@ -1,11 +1,19 @@
 # CTF Infrastructure Walkthrough
```

Comment:
should we mention kCTF (the name of the project) here or google cloud? to clarify what this walkthrough is about. wdyt/?

Reply:
Agree, this makes sense to add. I made some changes here at the beginning. Does it make sense?